conf/app.routes

```diff
@@ -1,3 +1,4 @@
 # microservice specific routes
 
-GET         /initiatives                               uk.gov.hmrc.platforminitiatives.controllers.PlatformInitiativesController.allInitiatives
\ No newline at end of file
+GET         /initiatives                               uk.gov.hmrc.platforminitiatives.controllers.PlatformInitiativesController.allInitiatives
+GET         /initiatives/:team                         uk.gov.hmrc.platforminitiatives.controllers.PlatformInitiativesController.teamInitiatives(team)
```

Comment:
wonder if the URL should be `teams/:team/initiatives` since it's typical to use the id of the resource type to navigate (`initiatives/:initiativeId`) , documents the role of the path-param and avoids collisions

Reply:
Amended to follow the URL convention above